tools/cockpit.spec

```diff
@@ -52,6 +52,10 @@ BuildRequires: docbook-style-xsl
 BuildRequires: keyutils-libs-devel
 BuildRequires: dbus-devel
 BuildRequires: glib-networking
+
+# There is a segfault in g_source_add_child_source that was fixed in
+# 2.37.3-2-g1d5c815
+BuildRequires: glib2-devel >= 2.40
```

Comment:
Hmm, it looks like configure.ac does do an explicit check for 2.37.4. That would help ensure that it *fails* if the buildroot has an older version, but doesn't force it to be pulled in if your build system happens to have the older version. (And for things like mock, it will fail earlier as the build-dep installation will fail instead of downloading and installing the older packages, then running configure to fail).

We should probably make sure these are the same value though. The upstream change appeared to happen in 2.37.3, but I'm fine with making this patch require 2.37.4 instead to match the existing configure.ac check. It's unlikely to make a difference.